src/components/test-summary/test-summary.ts

```diff
@@ -1,41 +1,33 @@
 import { Component } from '@angular/core';
 import { IFaultSummary, FaultTitle } from './interfaces/IFaultSummary';
+import { FaultStoreProvider } from '../../providers/fault-store/fault-store';
 
 @Component({
   selector: 'test-summary',
   templateUrl: 'test-summary.html'
 })
 export class TestSummaryComponent {
-  faultSummaries: IFaultSummary[] = [
-    {
-      title: FaultTitle.Dangerous,
-      total: 2,
-      faults: [
-        { name: 'Positioning - Lane Discipline', total: 1 },
-        { name: 'Response to signs - Other Road Users', total: 1 }
-      ]
-    },
-    {
-      title: FaultTitle.Serious,
-      total: 3,
-      faults: [
-        { name: 'Judgement - Cutting Corners', total: 1 },
-        { name: 'Awareness / Planning', total: 1 },
-        { name: 'Overtaking', total: 1 }
-      ]
-    },
-    {
-      title: FaultTitle.DriverFaults,
-      total: 8,
-      faults: [
-        { name: 'Judgement - Cutting Corners', total: 2 },
-        { name: 'Awareness / Planning', total: 1 },
-        { name: 'Overtaking', total: 1 },
-        { name: 'Positioning - Lane Discipline', total: 3 },
-        { name: 'Response to signs - Other Road Users', total: 1 }
-      ]
-    }
-  ];
+  testResult: string;
+  faultSummaries: IFaultSummary[];
 
-  constructor() {}
+  constructor(private faultStore: FaultStoreProvider) {
+    this.testResult = this.faultStore.getTestResult();
+    this.faultSummaries = [
```

Comment:
Looks like we're calling getTestResults() to populate the fault totals below. This is probably ok for Alpha but should we be utilising the store more for this kind of data. (See below comment)

Reply:
Perhaps it's just badly named? I could split it into two methods, one which returns 'Pass' or 'Fail' and one called `calculateTestReport()` which populates the relevant properties on the fault store?

Reply:
Yeah, two methods might be better. 👍